addon/authorizers/base.js

```diff
@@ -4,9 +4,7 @@ import Ember from 'ember';
   The base for all authorizers. __This serves as a starting point for
   implementing custom authorizers and must not be used directly.__
 
-  __The authorizer preprocesses all XHR requests__ (except ones to 3rd party
-  origins, see
-  [Configuration.crossOriginWhitelist](#SimpleAuth-Configuration-crossOriginWhitelist))
+  __The authorizer preprocesses all XHR requests__
```

Comment:
This isn't actually correct anymore neither but a big part of the docs needs to be updated anyway.